client/sammantha.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sammantha Sasenick</title>
+    <!-- Semantic UI -->
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" crossorigin="anonymous"></script>
```

Comment:
No need to load in js you're not using! Same with the semantic.min.js below, and jQuery at the bottom of the file.